### PR TITLE
Feature test the shopping list delete

### PR DESCRIPTION
**Pull request by SimonChenboYi**
co-authored-by: Carly <carly.jenkinson65@googlemail.com>

### CODE REVIEW

**Review by Nandini0206 — APPROVED** [review]
all tests look good :)

**Review by tsankhalpara — APPROVED** [review]
Looks great

File: DinnergyUITests/ShoppingListItemFeatureTest.swift [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    func testDeleteItem() {
+        let app = XCUIApplication()
+        
+        addBananas(number: "3")
+        
+        app.buttons["Recipes"].tap()
+        app.tables.element(boundBy: 0).cells.element(boundBy: 0).tap()
+        
+        let scrollViewsQuery = app.scrollViews
+        let veganBananaBreadElement = scrollViewsQuery.otherElements.containing(.staticText, identifier:"Vegan Banana Bread").element
```
Comment (jordantomiko) [inline DinnergyUITests/ShoppingListItemFeatureTest.swift:29]: Nice!!

**Review by jordantomiko — APPROVED** [review]
Well done Carly & Simon - these tests look like they must have been complicated to write but you've done a great job! :D

**Review by ocripps24 — APPROVED** [review]
Well done on extending the coverage with a complex test

**Pull request merged**